Purchases/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -78,6 +86,11 @@ class PurchasesOrchestrator {
         self.identityManager = identityManager
         self.receiptParser = receiptParser
         self.deviceCache = deviceCache
+        self.manageSubscriptionsModalHelper = manageSubscriptionsModalHelper
+        self.beginRefundRequestHelper = beginRefundRequestHelper
+        if #available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *) {
```

Comment:
We're going to use SK2 if it's available, otherwise we'll use SK1- as in: we're not going to all folks to decide what they want to use, right?

Reply:
well, I'd like to let them decide with a `shouldUseSK2WhenAvailable` flag before launch. Not sure it's needed for the first beta though

Reply:
https://app.shortcut.com/revenuecat/story/11122/feature-flag-shouldusesk2whenavailable